scripts/e2e/e2e-datadog-logs/test.yaml

```
@@ -0,0 +1,33 @@
+features:
+- e2e-tests-datadog
+
+test: "e2e"
+
+test_filter: "datadog::logs::"
+
+runner:
+  env:
+    EXPECTED_LOG_EVENTS: '1000'
+    VECTOR_RECEIVE_PORT: '8081'
+    FAKE_INTAKE_AGENT_ENDPOINT: 'http://fakeintake-agent:80'
+    FAKE_INTAKE_VECTOR_ENDPOINT: 'http://fakeintake-vector:80'
+
+matrix:
+  # validate against the latest nightly and also stable v6 and v7
+  version: ['latest', '6', '7']
```

Small nit - it would be helpful to specify that these versions are for the agent.
```suggestion
  agent_version: ['latest', '6', '7']
```